bentoml/cli/bento_management.py

```diff
@@ -172,23 +173,21 @@ def list_bentos(
         _print_bentos_info(result, output)
 
     @cli.command(
-        help='Delete bentos. To delete a bento service use "--name" and '
-        '"--version" options: "bentoml delete --name IrisClassifier '
-        '--bento-version 0.1.0" or "--tag" option: "bentoml delete --tag '
-        'IrisClassifier:v0.1.0". To delete multiple bentos, use "--name", "--tag", '
-        'and/or "--labels" to filter. To delete all bento services use "--all" option'
+        help='Delete bentos. Delete individual bento bundle: "bentoml delete '
```

Comment:
"Delete service bundles saved in target YataiService. When the `--yatai-url` option is not specified, it will use local Yatai by default."